lib/cli/Cargo.toml

```diff
@@ -24,19 +24,12 @@ doc = false
 required-features = ["headless"]
```

Comment:
The changes in this file are not needed any more.

Reply:
ok